src/olympia/users/models.py

```diff
@@ -1385,7 +1385,7 @@ def is_expired(self):
 
     @property
     def is_timedout(self):
-        return self.created + timedelta(seconds=30) < datetime.now()
+        return self.created + timedelta(minutes=1) < datetime.now()
```

Comment:
Per our discussion, let's bump to 10 minutes. We know from testing it can take around 7 minutes for the email to appear in the list, even if it has been sent faster than that.

Reply:
Done.